packages/core/data-transfer/src/strapi/providers/local-destination/index.ts

```diff
@@ -156,10 +156,14 @@ class LocalStrapiDestinationProvider implements IDestinationProvider {
       `uploads_backup_${Date.now()}`
     );
 
-    await fse.move(assetsDirectory, backupDirectory);
-    await fse.mkdir(assetsDirectory);
-    // Create a .gitkeep file to ensure the directory is not empty
-    await fse.outputFile(path.join(assetsDirectory, '.gitkeep'), '');
+    try {
+      await fse.move(assetsDirectory, backupDirectory);
+      await fse.mkdir(assetsDirectory);
+      // Create a .gitkeep file to ensure the directory is not empty
+      await fse.outputFile(path.join(assetsDirectory, '.gitkeep'), '');
```

Comment:
I'm ok with it like this, but it's possible (albeit unlikely) that the `move` works but the others fail. It might be nice to split this into two or three try/catch blocks so we know exactly which part went wrong.

Reply:
🤔 Can you give me an example of a situation that you came up with? Or is it to prevent? Because the issue that it's solving is a permissions issue, so I'd say that it would throw he same error in the three lines, wdyt?

Reply:
I was thinking about a case where the `move` works but then the `mkdir` throws, but the rollback should handle that case fine. And the messaging doesn't really need to be that specific.